src/src_main/crypto.c

```diff
@@ -21,6 +21,8 @@
 */
 #include "crypto.h"
 
+
+#include <gcrypt.h> // GCRY acs/ecs nums
```

Comment:
The libgcrypt interface should be the only thing that depends on libgcrypt -- the libgcrypt dependency should not be a core CryptoLib dependency. If you import libgcrypt here, then CryptoLib must have libgcrypt in order to be built and violates that idea. We should have our own enums that map to the libgcrypt enums when using the libgcrypt interface, or the KMC Crypto algorithms when using that  interface.